### PR TITLE
made CC 0.4 and CC 0.4 compatible with Ultimaker PLA

### PR DESCRIPTION
**Pull request by suppomi**
Copied CC 0.4 and CC 0.4 settings from GENERIC PLA to Ultimaker PLA materials

We are not sure if the PR #172 is ok only for GENERIC materials. where PLA we also have Ultimaker materials.
https://github.com/Ultimaker/fdm_materials/pull/172

### CODE REVIEW

**Comment by Ghostkeeper** [comment]
Note: We've been asked to ignore this PR for now. Suppomi and co. are working on a different fix. I'll convert this to a draft for that reason, to indicate it's not done yet.

**Comment by suppomi** [comment]
I do not have any further change for this branch.
In case if Cura team need this in next Cura release, firmware need to sync with Cura release.

File: ultimaker_pla_orange.xml.fdm_material [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+<fdmmaterial xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura" version="1.3">
     <metadata>
         <name>
             <brand>Ultimaker</brand>
             <material>PLA</material>
             <color>Orange</color>
         </name>
         <GUID>d9549dba-b9df-45b9-80a5-f7140a9a2f34</GUID>
-        <version>26</version>
+        <version>2</version>
```
Comment (shanx) [inline ultimaker_pla_orange.xml.fdm_material:10]:
```suggestion
        <version>27</version>
```

**Review by shanx — CHANGES_REQUESTED** [review]
spotted a typo

File: ultimaker_pla_orange.xml.fdm_material (hunk 1 above)
Comment (bryanfoley) [reply ultimaker_pla_orange.xml.fdm_material:10]: Sharp :eyes:

**Comment by atrebelo** [comment]
**_PLEASE DO NOT APPROVE OR MERGE THIS PR UNTIL FURTHER NOTICE_**

Message from Stijn in slack from Friday 25 June 2021

> Stijn Arntz  3 days ago

> The main reason why we have a compatibility of the CC06 core with Generic PLA is for third party materials.
> So that this profile can be used as a base for filled third party PLA's.
> Based on this it is not necessary to change the compatibility for UM PLA's right now.
> However, we would like to have a compatibility with CC04 and Generic PLA which is currently there right? <Confirmed!>

> When Sarita is back she can check how it should work for UM PLA's in my opinion.

**Comment by Ghostkeeper** [comment]
Sarita referred further on to @THeijmans, who is now on vacation. Monday he'll be back apparently.

**Review by Ghostkeeper — APPROVED** [review]

**Comment by Ghostkeeper** [comment]
As discussed, it is indeed expected behaviour that Ultimaker PLA is supported in the same configurations as Generic PLA and should behave the same, so we've merged this change.

**Pull request merged**